### PR TITLE
Get/SetComplete Test Pass

### PR DESCRIPTION
**Pull request by EloisePrasad**
Passing Tests for getComplete() and setComplete() methods

### CODE REVIEW

File: src/test/java/ToDoTest.java [hunk 1]
```diff
@@ -24,6 +24,22 @@ public void setContentisCorrect(){
...
+    public void defaultCompleteIsFalse() {
+        assertFalse(toDo.getComplete());
+    }
+
+    @Test
+    public void canSetNotComplete(){
+        toDo.setComplete(true);
+        toDo.setComplete(false);
+        assertFalse(toDo.getComplete());
+    }
```
Comment (sebastianchristopher) [inline src/test/java/ToDoTest.java:44]: Consider writing an @After method (public void tearDown) setting ToDo to null.

File: src/main/java/ToDo.java [hunk 2]
```diff
@@ -1,13 +1,23 @@
 public class ToDo {
     String content;
+    boolean complete;
```
Comment (sebastianchristopher) [inline src/main/java/ToDo.java]: These should probably be private (also String content on line 2) as we don't necessarily want them to be immediately accessible outside the class (hence our getter and setter methods).

**Review by sebastianchristopher — CHANGES_REQUESTED** [review]
👍
Couple of suggested changes.

**Comment by EloisePrasad** [comment]
Changes made in new commit

**Pull request closed without merging**